src/query/interpreter.hpp

```diff
@@ -49,6 +49,8 @@ namespace memgraph::query {
 
 inline constexpr size_t kExecutionMemoryBlockSize = 1UL * 1024UL * 1024UL;
 
+constexpr std::string_view kAsterisk = "*";
```

Comment:
There already exists `kAsterisk` in `src/auth/models.hpp` I would think about reusing it, but to do that, it makes sense to move it someplace else. I don't have a strong opinion on this, feel free to ignore it.